packages/Sandblocks-Core/SBEditor.class.st

```diff
@@ -134,7 +151,7 @@ SBEditor >> addToMultiSelection: aBlock [
 { #category : #accessing }
 SBEditor >> artefacts [
 
-	^ scroll scroller submorphs select: [:s | s isSandblock and: [s isArtefact]]
+	^ scroll scroller submorphs select: [:s | (s isSandblock or: s isConnector) and: [s isArtefact]]
```

Comment:
missing `[]` after the `or:` (negligibly better performance but it is a convention, so...)

Reply:
Ah, do you need to change `artefacts` in this manner? Some other code may break if you return non-SBBlock classes here. Do you maybe want to introduce a new selector for these, such as `forceAffectedMorphs` or something catchier?